src/components/details-page/components/details-page/index.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {
+  memo,
+  FC,
+  PropsWithChildren,
+  useEffect,
+  Children,
+  isValidElement
+} from 'react';
+import { Link } from 'react-router-dom';
+
+import translations from '../../../../lib/localization';
+import { getTranslateText as translate } from '../../../../lib/translateText';
+
+import withReferenceData, {
+  Props as ReferenceDataProps
+} from '../../../with-reference-data';
+
+import Banner from '../banner';
+import ContentSection from '../content-section';
+
+import OpenAccessIcon from '../../../../images/icon-access-open-md.svg';
+
+import SC from './styled';
+
+import { LosTheme, EuTheme } from '../../../../types';
+import { Entity } from '../../../../types/enums';
+
+interface Theme {
+  id: string;
+}
+
+interface EnrichedTheme extends Partial<LosTheme>, Partial<EuTheme> {}
+
+interface Props extends ReferenceDataProps {
+  entity: Entity;
+  title: string;
+  publisher: string;
+  lastPublished: string;
+  isAuthoritative: boolean;
+  isOpenData: boolean;
+  themes: Theme[];
+}
+
+const rootPaths = {
+  [Entity.DATASET]: '',
+  [Entity.DATA_SERVICE]: 'apis',
+  [Entity.CONCEPT]: 'concepts',
+  [Entity.INFORMATION_MODEL]: 'informationmodels'
+};
+
+const DetailsPage: FC<PropsWithChildren<Props>> = ({
+  entity,
+  title,
+  publisher,
+  lastPublished,
+  isAuthoritative,
+  isOpenData,
+  themes = [],
+  referenceData: { los: losThemes, themes: euThemes },
+  referenceDataActions: { getReferenceDataRequested: getReferenceData },
+  children
+}) => {
+  useEffect(() => {
+    const appRoot = document.querySelector('#root > div');
```

Comment:
Jeg mener vi i dag setter enten 'theme-fdk' eller 'theme-nap', og at det på den måten velges to ulike temafarger. Men nå velger du theme enten dataset/api/concept/informationmodel? Har vi fortsatt en mulighet for å vise NAP-theme for detaljside datasett?

Reply:
Ja, det har vi. Både FDK og NAP themes har sine egne farger for dataset, api, concept og informationmodel.